README.md

```diff
@@ -10,22 +10,23 @@ Due to the fact that Go 1.11 module versions have semantic meaning and don't nec
 
 | libgit2 | git2go        |
 |---------|---------------|
-| main    | (will be v34) |
-| 1.3     | v33           |
+| main    | (will be v35) |
+| 1.5     | v34           |
+| 1.3     | v34           |
```

Comment:
Should this be v33 since v34 is for 1.5?

Reply:
it should!

```suggestion
| 1.3     | v33           |
```